transaction/std/src/transaction_builder.rs

```diff
@@ -57,6 +57,21 @@ pub struct TxOutContext {
     pub shared_secret: RistrettoPublic,
 }
 
+/// Signing data for external library
+#[derive(Debug)]
+pub struct TransactionViewOnlySigningData {
```

Comment:
Since this is more than just the signing data (it has the bulk of the transaction - TxPrefix which is the entire transaction sans the signature), I am leaning towards renaming it to `ViewOnlyUnsignedTx`. Thoughts?